rules/solid/tests/Rector/Variable/MoveVariableDeclarationNearReferenceRector/Fixture/variable_in_closure.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+function variableInClosure()
+{
+    $closure = function () {
+        $var = 1;
```

Comment:
So the problem is that removing assing does not remove this line?

Reply:
In case of stmts (each function like has many stmts), you might need to iterate these stmts and unset them.

I'll try it and commit it to the PR

Reply:
Both the removing of assign and the adding of the assign somewhere else doesn't do anything.